gc.rb

```diff
@@ -193,7 +193,10 @@ def self.count
   # - +:time+:
   #   The total time spent in garbage collections (in milliseconds).
   # - +:heap_allocated_pages+:
-  #   The total number of +:heap_eden_pages+ + +:heap_tomb_pages+.
+  #   The total number of allocated pages.
+  # - +:heap_empty_pages+:
+  # - +:heap_allocated_pages+:
```

Comment:
`heap_allocated_pages` is listed twice, does this line belong?

<img width="788" height="186" alt="image" src="https://github.com/user-attachments/assets/3a6a61c1-c4aa-4b9a-956d-4235070a6dee" />